### PR TITLE
Remove adb sdklib

### PR DESCRIPTION
**Pull request by cesarferreira**

### CODE REVIEW

**Comment by luispereira** [comment]
I will take advantage of this PR for refactoring and cleaning up the code regarding the removal of  adb lib

**Comment by luispereira** [comment]
I think its all for now, any issue you might have please let me know

File: lib/dryrun/android_project.rb [hunk 1]
```diff
@@ -150,68 +157,68 @@ def sample_project
...
+    DryrunUtils.run_adb("shell pm uninstall #{@package}")
+  end
 
-    def get_execution_line_command(path_to_sample)
-      manifest_file = get_manifest(path_to_sample)
+  def get_execution_line_command(path_to_sample)
+    manifest_file = get_manifest(path_to_sample)
 
-      if manifest_file.nil?
-        return false
```
Comment (cesarferreira) [inline lib/dryrun/android_project.rb:168]: the `return` is optional in ruby, you could just wrote `false` or

``` ruby
return false if manifest_file.nil?
```

Comment (luispereira) [reply lib/dryrun/android_project.rb:168]: hey thanks for the tip, will use it on the future. thanks

**Pull request merged**